### ISSUE TITLE
Entered value in column field reverts to default when user clicks out of it. Only works by Tabbing

### ISSUE THREAD

**Issue by tdoshin** [microsoft/azuredatastudio#20256]
- Azure Data Studio Version: 1.38
- OS Version:

Steps to Reproduce:

1. Create new Table
2. Edit name field in column
3. Click out of column into anywhere else on TD

Expected Result:

Input entered in column remains intact 

Actual Result: 

Input reverts to default value

Does this issue occur when all extensions are disabled?: No 

@alanrenmsft  is this related to the work @smartguest has been working on?

**Comment by smartguest** [comment]
@tdoshin Not quite, my work only affects when you right click on an existing table in the object explorer and press edit data.

**Comment by caohai** [comment]
Here is what I'm seeing with latest release. Looks like the input is preserved when I click out. Is this the same steps you used to repro the issue?
![rename](https://user-images.githubusercontent.com/21186993/183158386-d9c97a70-f9fd-4bbb-9917-b4726ce0047a.gif)

Version: 1.38.0 (user setup)
Commit: dff93c4736269ef2a7e8a225737a0d8354bf0246
Date: 2022-07-26T23:56:16.217Z
VS Code: 1.62.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19044

**Comment by tdoshin** [comment]
@caohai Seems to be working now. Not sure why it wasn't the other day when @erinstellato-ms and I were testing

**Comment by tdoshin** [comment]
Hey @caohai and @alanrenmsft. See, I was able to reproduce again. See GIF. Here are the steps I took 

1.) Clicked inside existing column with default name and changed to 'columntimi' 

2.) Navigate to "+New Column" using mouse. Be sure to do this without typing out of the Tab field 


![Column name reversing](https://user-images.githubusercontent.com/72947001/184690590-44a60c19-10b8-4f5b-ab73-419d8533b142.gif)
